container/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# This dockerfile creates a container of a truly basic DIRAC installation which contains only the DIRAC externals.
+# The sole purpose of this container is for developing (and testing) DIRAC "stuff that run",
+# meaning DIRAC components like agents and services. Instructions in ...
+
+# THIS IS A WORK IN PROGRESS
+# TODO:
+#   verify about certificates
+#   correct openssl_config URL location
+#   lcgBundle now fix version
+#   start mySQL
+#   install/start ElasticSearch (?)
```

Comment:
I would say that if you want to use external services this way, you are better off using docker-compose